### ISSUE TITLE
How to cite

### ISSUE THREAD

**Issue by agussa** [polo8214/Brain-tumor-segmentation#10]
This is not a problem. Just a question. your work is great! It inspired me to modify to suit my problem. How to cite your work ?

Thanks.

**Comment by Kelvinyu1117** [comment]
Same question for citation.

@agussa 
I would like to ask do you know how to retrain the network?
How can I read all the training data as x, and label as y?

**Comment by agussa** [comment]
if you have enough RAM/ GPU memory than you can do this
> How can I read all the training data as x, and label as y?

But, if you don't have enough RAM/GPU RAM, then it better to use keras.sequence as generator or tf.keras.sequence to fetch one by one and feed to network, or per-batch as you specify it.

**Comment by Kelvinyu1117** [comment]
Do you have any code samples?
I am not quite familiar with the generator...
Can I use the function create_data in the notebook to read it?
